Cocktails9/Cocktails9/Manager/NetworkManager.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+
+class NetworkManager {
+    
+    // Fetch alcoholic drinks (the default method)
+    static func fetchDrinks() async {
+        guard let url = URL(string: "https://www.thecocktaildb.com/api/json/v1/1/filter.php?a=Alcoholic") else { return }
```

Comment:
Declare private variable baseUrl and endpoints in NetworkManager and use it inside functions like this
`URL(string: baseUrl + alcoholic)`